client/src/app/site/deployment-center/deployment-center-setup/wizard-logic/deployment-center-state-manager.ts

```diff
@@ -463,13 +463,32 @@ export class DeploymentCenterStateManager implements OnDestroy {
     this._localStorageService.setItem(storedPermissionItem.id, storedPermissionItem);
   }
 
```

Comment:
was the method getVstsPassthroughHeaders not used in this file?

Reply:
No, it is not used anywhere